doc/users/colormapnorms.rst

```diff
@@ -0,0 +1,123 @@
+.. _colormapnorm-tutorial:
+
+Colormap Normaliztions 
+================================
+
+Objects that use colormaps usually linearly map the colors in the
+colormap from data values ``vmin`` to ``vmax``.  For example::
+
+    pcm = ax.pcolormesh(x, y, Z, vmin=-1., vmax=1., cmap='RdBu_r')
+
+will map the data in *Z* linearly from `-1` to `+1`, so *Z=0* will
+give a color at the center of the colormap *RdBu_r* (white in this
+case).  
+
+Normalizations are defined as part of :func:`matplotlib.colors`
```

Comment:
Can you add a sentence or two like "color mapping is a two step process, first the data is mapped to the range [0, 1] and then those values are mapped to RGB.  How the mapping to the unit range happens is called the normalization in matplotlib."